src/settings.rs

```diff
@@ -111,41 +119,41 @@ macro_rules! create_keybind {
             description: $description,
             mutable: true,
             serializable: true,
-            default: &|| VirtualKeyCode::$keycode as i64,
+            default: &|| VirtualKeyCode::$keycode as u8,
         }
     };
 }
 
-pub const CL_KEYBIND_FORWARD: console::CVar<i64> =
+pub const CL_KEYBIND_FORWARD: console::CVar<u8> =
```

Comment:
Are you sure, that all key ids are only in u8 range? And not for example u16 range?

Reply:
Keycodes are always values in u8 range. This also can be checked by looking at `VirtualKeyCode` enum even though it uses `repr(u32)`. It is impossible to have keyboard codes above 255. For reference, here's structure used to report virtual keycodes by Windows:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-keybdinput